src/main/kotlin/io/kotlintest/matchers/ComparableMatchers.kt

```diff
@@ -0,0 +1,49 @@
+package io.kotlintest.matchers
+
+@Deprecated("Use `value shouldBe lt(x)` or `value should beLessThan(x)`")
```

Comment:
Primarily keeping to conventions (as like `Int` or `Long` matchers), as they have the same surface behaviours. Also with the added benefit of helping users coming from other libraries and/or `v1`.